src/components/neumorphic/LightSourceSelector.tsx

```diff
@@ -0,0 +1,47 @@
+import { MouseEvent } from 'react';
+import style from '../../style/sass/components/neumorphic/configuration.module.scss';
+
+interface LightSourceSelectorProps {
+  lightSource: number;
+  disabled?: boolean;
+  onDirectionChanged: (name: number) => void;
+}
+
+const LightSourceSelector = ({
+  lightSource,
+  onDirectionChanged,
+  disabled,
+}: LightSourceSelectorProps) => {
+  const handleClick = (event: MouseEvent<HTMLButtonElement>) => {
+    const direction = Number((event.target as HTMLInputElement).name);
+    onDirectionChanged(direction);
+  };
+  return (
+    <div className={`${style.row} ${style.label}`} style={{ margin: '12px 0px 8px' }}>
+      <div className={`${style.lightArrows} ${disabled ? style.disabled : ''}`}>
+        <button
+          className={`${style.arrow} ${lightSource === 1 ? style.active : ''}`}
+          onClick={handleClick}
+          name="1"
+        ></button>
```

Comment:
Mejorar el codigo repetido y la accesibilidad de todos los botones haciendo algo como esto:
```suggestion
  {directions.map((direction) => (
    <button
      key={direction}
      className={`${style.arrow} ${lightSource === direction ? style.active : ''}`}
      onClick={handleClick}
      name={`${direction}`}
    ></button>
  ))}
```